src/entities/currency.ts

```diff
@@ -33,5 +40,6 @@ export class Currency {
   }
 }
 
-const CAVAX = Currency.CAVAX
+//$ const CAVAX = Currency.CAVAX
+const CAVAX = Currency.CURRENCY
```

Comment:
I think this change will break interface / components. so we need to release this sdk version as major version and also update interface/components

Reply:
yes these changes are breaking and we have the interface/ components code that fits this version ready to PR once the SDK is published. this is probably better anyway to release a major version. 
this interface branch works with the 2.0 sdk https://github.com/wagmiog/interface/tree/implement-multichain
and we have a live interface here: https://quirky-kowalevski-0e801a.netlify.app/#/beta/airdrop